### ISSUE TITLE
Cannot install django-ses on Ubuntu 18.04: version not found

### ISSUE THREAD

**Issue by wrvdklooster** [django-ses/django-ses#251]
Can not install django-ses 3.1.0 on Ubuntu 18.04 system. Locally on MacOS no issue.

On MacOS:
```
$ pip index versions django-ses
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
django-ses (3.1.0)
Available versions: 3.1.0, 3.0.1, 3.0.0, 2.6.1, 2.6.0, 2.5.0, 2.4.0, 2.3.1, 2.3.0, 2.2.2, 2.2.1, 2.2.0, 2.1.1, 2.1.0, 2.0.0, 1.0.3, 1.0.2, 1.0.1, 1.0.0, 0.8.14, 0.8.13, 0.8.12, 0.8.11, 0.8.10, 0.8.9, 0.8.8, 0.8.7, 0.8.6, 0.8.5, 0.8.4, 0.8.3.1, 0.8.2, 0.8.1, 0.8.0, 0.7.1, 0.7.0, 0.6.0, 0.4.1, 0.4.0, 0.3.0, 0.2, 0.1, 0.8.3.linux-x86_64
  INSTALLED: 3.1.0
  LATEST:    3.1.0
```

On Ubuntu 18.04:
```
$ pip index versions django-ses
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
django-ses (2.6.1)
Available versions: 2.6.1, 2.6.0, 2.5.0, 2.4.0, 2.3.1, 2.3.0, 2.2.2, 2.2.1, 2.2.0, 2.1.1, 2.1.0, 2.0.0, 1.0.3, 1.0.2, 1.0.1, 1.0.0, 0.8.14, 0.8.13, 0.8.12, 0.8.11, 0.8.10, 0.8.9, 0.8.8, 0.8.7, 0.8.6, 0.8.5, 0.8.4, 0.8.3.1, 0.8.2, 0.8.1, 0.8.0, 0.7.1, 0.7.0, 0.6.0, 0.4.1, 0.4.0, 0.3.0, 0.2, 0.1, 0.8.3.linux-x86_64
  INSTALLED: 2.6.1
  LATEST:    2.6.1
```

**Comment by pcraciunoiu** [comment]
On Ubuntu 22.04, I don't see this issue.

Are you sure your pip index isn't cached? Try some tips from this comment https://github.com/pypa/pip/issues/4830#issuecomment-627632928

```
$ pip index versions django-ses
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
django-ses (3.1.0)
Available versions: 3.1.0, 3.0.1, 3.0.0, 2.6.1, 2.6.0, 2.5.0, 2.4.0, 2.3.1, 2.3.0, 2.2.2, 2.2.1, 2.2.0, 2.1.1, 2.1.0, 2.0.0, 1.0.3, 1.0.2, 1.0.1, 1.0.0, 0.8.14, 0.8.13, 0.8.12, 0.8.11, 0.8.10, 0.8.9, 0.8.8, 0.8.7, 0.8.6, 0.8.5, 0.8.4, 0.8.3.1, 0.8.2, 0.8.1, 0.8.0, 0.7.1, 0.7.0, 0.6.0, 0.4.1, 0.4.0, 0.3.0, 0.2, 0.1, 0.8.3.linux-x86_64
WARNING: You are 
```

**Comment by wrvdklooster** [comment]
Thanks for the fast reply!

Was in the end an issue with an incorrect Python version in our CI project.

Sorry for bothering you with this. Issue can be closed.
